Framework/test/testKafkaPoller.cxx

```diff
@@ -22,13 +22,32 @@
 TEST_CASE("test_kafka_poller")
```

Comment:
indeed we would need a kafka broker to run this test automatically... should we mark it as manual in CMake meanwhile?